cfme/cloud/tenant.py

```diff
@@ -101,12 +103,13 @@ def delete(self, cancel=False, from_details=True, wait=True):
             sel.handle_alert(cancel=cancel)
 
         # In both cases the flash message is the same
-        if not cancel:
+        if cancel:
```

Comment:
Yeah I actually thought about this and left it - It needs to be further down with the flash message assert. 'both cases' refers to from_details=True|False, and that's not clear here at all. thx.